.github/workflows/report-vulnerability-in-gh.yaml

```diff
@@ -0,0 +1,78 @@
+name: Report vulnerability issues in Github
+on:
+  workflow_call:
+    inputs:
+      issue-title:
+        description: The title of the issue to be created/edited
+        required: true
+        type: string
+      issue-labels:
+        description: A comma separated list of labels
+        required: false
+        type: string
+        default: "bug"
+      image-name:
+        description: "Name of the oci-image as saved in Dockerhub or in the docker cache.
+          It consists of <rock-name>:<rock-version>."
+        required: true
+        type: string
+
+jobs:
+  report-vulns:
+    name: Create or edit issues for reporting vulnerabilities
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install tools
+        run: |
+          sudo snap install gh
+          sudo snap install jq
+
+      - name: Generate image name
+        id: image-name
+        run: |
+          IMAGE_NAME=$(echo ${{ inputs.image-name }} | sed 's/\:/-/g')
+          echo "image-name=${IMAGE_NAME}" >> "$GITHUB_OUTPUT"
+
+      - name: Get issue number if exists
+        id: get-issue-number
+        run: |
+          export GH_TOKEN=${{ secrets.GH_TOKEN }}
+          EXPECTED_TITLE=$(echo "${{ inputs.issue-title }} ${{ inputs.image-name }}")
```

Comment:
This EXPECTED_TITLE is already using `inputs.image-name`, but right now the title would look like "Vulnerabilities found for jupyter-pytorch-full-1.8.0".

I'll make sure to pass the correct input for `image-name` so that the title gets rendered as "Vulnerabilities found for jupyter-pytorch-full:1.8.0"

Reply:
Please check the latest run of the workflow, we now have the title as you mentioned. See https://github.com/DnPlas/kubeflow-rocks/issues/69 for an example.

Reply:
Looks very good, did we run more than once to confirm this works as expected for editing existing issues too? If yes, you can resolve this comment

Reply:
Yes! [This](https://github.com/DnPlas/kubeflow-rocks/issues/71) is an example (:
Resolving the comment.